torch/linalg/__init__.py

```diff
@@ -2156,7 +2156,7 @@
     Consider using :func:`torch.linalg.tensorsolve` if possible for multiplying a tensor on the left
     by the tensor inverse, as::
 
-        tensorsolve(A, B) == torch.tensordot(tensorinv(A), B)
+        tensorsolve(A, B) == torch.tensordot(tensorinv(A), B)  # When B is a matrix
```

Comment:
It'd be good to write `linalg.tensorsolve` for consistency and edit the link to the function in the line above as well.

Also, what do you mean by "when `B` is a matrix? I think this one is slightly more subtle, as `B` is a tensor in general. Dos this happen when `ind=-1`?

Reply:
Yes, knuckle head move of me. `tensorsolve` is more nuanced. These statements are equal:

Reply:
And ind must be a positive integer according to the tensorinv docs its not possible to use negative indexing here

Reply:
Not sure what you think but my thougts are to update 2 lines tensorinv:
```
tensorsolve(A, B) == torch.tensordot(tensorinv(A), B)  # When B has shape A.shape[:B.ndim]
```

And
```
B (Tensor): tensor of shape :attr:`A`\ `.shape[:\ `:attr:`B`\ `.ndim]`.
```

Reply:
LGTM!